openapi.yaml

```diff
@@ -25,254 +37,677 @@ security:
+        - in: path
+          name: pizzaID
+          required: true
+          type: string
+          description: "ID of pizza to retrieve"
+      responses:
+        200:
+          description: "A single pizza"
+          schema:
+            $ref: "#/definitions/PizzaResult"
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
+    patch:
+      description: "Update a pizza recipe"
+      tags:
+        - pizza
+      parameters:
+        - in: path
+          name: pizzaID
           required: true
+          type: string
+          description: "ID of a pizza to retrive"
+        - in: body
+          name: attributes
+          description: "attributes of the pizza to be updated"
           schema:
-            $ref: '#/definitions/PetPostBody'
+            $ref: "#/definitions/PizzaAttributes"
       responses:
-        '201':
-          description: Pet successfully created
+        200:
+          description: "The updated pizza recipe"
           schema:
-            $ref: '#/definitions/PetResult'
-        '400':
-          $ref: '#/responses/400'
-        '409':
-          description: Conflict
+            $ref: "#/definitions/PizzaResult"
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
+    delete:
+      description: "Delete a pizza recipe"
+      tags:
+        - pizza
+      parameters:
+        - in: path
+          name: pizzaID
+          required: true
+          type: string
+          description: "ID of a pizza to retrive"
+      responses:
+        200:
+          description: "Returns true is the resource was deleted"
           schema:
-            $ref: '#/definitions/ErrorResult'
-        '500':
-          $ref: '#/responses/500'
-  /pets/{id}:
+            type: boolean
+            example: true
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
+
+  /pizzas/{pizzaID}/ingredients:
     get:
-      summary: Info for a specific pet
+      description: "Get the ingredients from the pizza with ID pizzaID"
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - pizza
+        - ingredient
       parameters:
-        - name: id
-          in: path
-          description: Unique Pet ID, UUID format
+        - in: path
+          name: pizzaID
           required: true
           type: string
+          description: "ID of a pizza to retrive"
       responses:
-        '200':
-          description: Successful response
+        200:
+          description: "A list of ingredients"
           schema:
-            $ref: '#/definitions/PetResult'
-        '404':
-          description: PetResource not found
+            $ref: "#/definitions/IngredientsResult"
+        404:
+          $ref: "#/responses/404"
+        500:
+          $ref: "#/responses/500"
+  /pizzas/{pizzaID}/ingredients/{ingredientID}:
+    patch:
```

Comment:
not totally clear if `put` or `patch` is the right method to use here -- I went with `patch` because I was thinking in terms of updating the pizza resource